README.md

```diff
@@ -37,16 +37,16 @@ Versions match. As an example, to get Petals Cockpit 0.20.0, just type in `docke
 ## Run parameters
 
 When running Petals Cockpit by a docker image, a default administrator user is added automatically:
-* user: `admin`
 * username: `admin`
+* user: `admin`
 * password: `admin`
 
 It is possible to change this user with the following parameters: 
 
 | Argument | Optional | Default | Description |
 | -------- | :------: | :-----: | ----------- |
-| COCKPIT_USER | yes | `admin` | The user's id, also his login. |
-| COCKPIT_USERNAME | yes | `admin` | The name under which the user will appear. |
+| COCKPIT_USERNAME | yes | `admin` | The user's id, also his login. |
```

Comment:
**USER_ID** would be more explicit then.

Reply:
`username` is the term used in cockpit for the user UID.